benchmarks/config.h

```diff
@@ -21,7 +21,8 @@
 
 struct AppConfig
 {
-    std::string input_file = "test_data/private/generic_tiff_000.tif";
+    std::string test_folder;
+    std::string test_file;
```

Comment:
We may want to revisit at some point and look at using [`std::filesystem::path`]( https://en.cppreference.com/w/cpp/filesystem/path ). Though that requires C++17 and we were already using `std::string`. So just a future note

Reply:
Thanks!
Created a issue for following up:
- https://github.com/rapidsai/cucim/issues/136